Parse/Parse.xcodeproj/project.pbxproj

```diff
@@ -8936,6 +8948,10 @@
 			baseConfigurationReference = 81C582DE1C3B0A40000063C6 /* Parse-iOS-Dynamic.xcconfig */;
 			buildSettings = {
 				CLANG_MODULES_AUTOLINK = YES;
+				CURRENT_PROJECT_VERSION = 1.18.0;
+				MARKETING_VERSION = 1.18.0;
+				SUPPORTS_MACCATALYST = NO;
+				TARGETED_DEVICE_FAMILY = 1;
```

Comment:
This suggests we only support iPhone. Should this be 1, 2?

Reply:
I guess we need to define our target naming convention better. When I see Parse-iOS I think iPhone only. Also I’m a bit confused by the Parse-iOS-Dynamic because on other frameworks they use that target naming convention to convey that it works on iOS, iPad OS, and Mac Catalyst,

Reply:
I would expect that the iOS build works on those 3 platforms too (assuming compatibility with Catalyst).

Reply:
Well essentially what happened with the release of iOS 13 is it removed the need to have a iOS specific AND an iOS dynamic. If you remember previously we where linking all the libraries specific to that OS depending on the target. Once you open this project Xcode11 it automatically adds catalyst support and it makes it where all system dependencies are linked automatically. Which is why we get these warnings a lot now 

```
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/Cocoa.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/Accounts.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/AppKit.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/Security.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/CFNetwork.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/CoreGraphics.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/Social.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/StoreKit.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/AudioToolbox.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/QuartzCore.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/CoreLocation.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/AudioToolbox.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/SystemConfiguration.framework' in workspace 'Parse.xcworkspace'.
--- xcodebuild: WARNING: Unable to open project file '/System/Library/Frameworks/Security.framework' in workspace 'Parse.xcworkspace'.
```
This is because all system frameworks now link dynamically for every target family.